ramp-database/ramp_database/tools/frontend.py

```diff
@@ -140,3 +145,27 @@ def is_user_signed_up(session, event_name, user_name):
             (event_team.is_active and event_team.approved)):
         return True
     return False
+
+
+def did_user_signed_up(session, event_name, user_name):
```

Comment:
Maybe let's call this, `is_user_sign_up_requested` to make it more consistent with the `is_user_signed_up`?